.github/workflows/cypress.yaml

```
@@ -0,0 +1,148 @@
+              docker exec $CONTAINER_NAME apt-get update && docker exec $CONTAINER_NAME apt-get install -y curl
+            }
+            RESPONSE=$(docker exec $CONTAINER_NAME curl -s -X POST http://127.0.0.1:4000/graphql -H "Content-Type: application/json" -d '{"query":"{__typename}"}' 2>/dev/null || echo "Connection failed")
+            if echo "$RESPONSE" | grep -q '__typename'; then
+              echo "GraphQL endpoint is available!"
+              exit 0
+            fi
+            echo "GraphQL endpoint not ready. Retrying in 2 seconds..."
+            sleep 2
+          done
+          echo "GraphQL endpoint did not become available within the expected time."
+          exit 1 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)